bindings/python/pyproject.toml

```diff
@@ -36,6 +36,7 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
```

Comment:
Is this really the only change we need to make to support 3.13? Wow!

Reply:
We had been testing in GitHub Actions, but now in EVG as well.  :smile: